docs/troubleshooting.md

```diff
@@ -0,0 +1,16 @@
+# Troubleshooting
+
+## Installation
+
+If when installing using
+```
+kubectl create -f kubeless.yaml --namespace kubeless
+```
+gives the following error:
+```
+customresourcedefinition "functions.k8s.io" created
+error: error validating "kubeless.yaml": error validating data: unknown object type schema.GroupVersionKind{Group:"", Version:"v1", Kind:"Service"}; if you choose to ignore these errors, turn validation off with --validate=false
+```
+
+You probaly have an older version of Kubernetes. Make sure
+you are using at least version `1.7.5`
```

Comment:
Oh, interesting, I didn't realise CRDs didn't work v1.7.[0-4]. Do you have a pointer to a bug/feature/release-note/discussion that links to the relevant change in v1.7.5?

Reply:
Probably an issue with my setup. After testing, it works fine on 1.7.0+